analyzer-java/src/main/java/com/infosupport/ldoc/analyzerj/AnalysisVisitor.java

```diff
@@ -31,7 +31,7 @@
 import com.github.javaparser.resolution.declarations.ResolvedAnnotationDeclaration;
 import com.github.javaparser.resolution.types.ResolvedType;
 import com.infosupport.ldoc.analyzerj.descriptions.*;
-import com.infosupport.ldoc.analyzerj.helperMethods.CommentHelperMethods;
+import com.infosupport.ldoc.analyzerj.helper_methods.CommentHelperMethods;
```

Comment:
If we're renaming this anyway, probably best to follow [Google style](https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names), so no underscores.

Reply:
i see then sure i will follow google_style.